sdk/core/azure-core/test/ut/log_policy.cpp

```diff
@@ -0,0 +1,262 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/core/diagnostics/logger.hpp>
+#include <azure/core/http/policies/policy.hpp>
+#include <azure/core/internal/http/pipeline.hpp>
+
+#include <gtest/gtest.h>
+
+using Azure::Core::Diagnostics::Logger;
+using Azure::Core::Http::Policies::LogOptions;
+
+namespace {
+void SendRequest(LogOptions const& logOptions)
+{
+  using namespace Azure::Core;
+  using namespace Azure::Core::IO;
+  using namespace Azure::Core::Http;
+  using namespace Azure::Core::Http::_internal;
+  using namespace Azure::Core::Http::Policies;
+  using namespace Azure::Core::Http::Policies::_internal;
+
+  class TestTransportPolicy : public HttpPolicy {
+  public:
+    std::unique_ptr<HttpPolicy> Clone() const override
+    {
+      return std::make_unique<TestTransportPolicy>(*this);
+    }
+
+    std::unique_ptr<RawResponse> Send(Request&, NextHttpPolicy, Context const&) const override
+    {
+      constexpr uint8_t const responseBody[] = "Response Body";
+      constexpr uint8_t const responseBodyStream[] = "Request Body Stream";
+
+      auto response = std::make_unique<RawResponse>(1, 1, HttpStatusCode::Ok, "OKAY");
+
+      response->SetBody(std::vector<uint8_t>(responseBody, responseBody + sizeof(responseBody)));
+
+      response->SetBodyStream(
+          std::make_unique<MemoryBodyStream>(responseBodyStream, sizeof(responseBodyStream) - 1));
+
+      return response;
+    }
+  };
+
+  constexpr uint8_t const requestBodyStream[] = "Request Body Stream";
```

Comment:
static?

Reply:
Does not really matter for constexpr, but why not, I'll add it.